client/index.html

```diff
@@ -7,6 +7,11 @@
     <script src="https://kit.fontawesome.com/86410157d4.js" crossorigin="anonymous"></script>
   </head>
   <body>
+    <div class="covid-banner">
+      <div class="covid-text">
```

Comment:
Doesn't seem necessary, but wonder if it makes practical/good practice to wrap any of the upper divs in <header> tags?

Reply:
@claire-melbourne so the only reason I added in the additional div for covid text, was for some reason the CSS stylings I wanted were not attaching to the covid banner div class. The only way to get the css to be passed to it properly, was by nesting another div inside. I'm not really sure what exactly is going on, but that was my workaround.